core/lib/asset-saver.js

```diff
@@ -99,6 +152,36 @@ function stringifyReplacer(key, value) {
   return value;
 }
 
+/**
+ * @param {LH.UserFlow.FlowArtifacts} flowArtifacts
+ * @param {string} basePath
+ * @return {Promise<void>}
+ */
+async function saveFlowArtifacts(flowArtifacts, basePath) {
```

Comment:
A comment here or in loading code describing the file structure would be helpful.